tool/tsh/proxy_test.go

```diff
@@ -36,10 +35,7 @@ import (
 func TestProxySSHDial(t *testing.T) {
 	createAgent(t)
 
-	os.RemoveAll(profile.FullProfilePath(""))
```

Comment:
This was the only use of `os` in the file, the test now doesn't compile because of
```tool/tsh/proxy_test.go:20:2: imported and not used: "os"```.